spec/controllers/saml_idp_controller_spec.rb

```diff
@@ -344,10 +344,6 @@
         generate_saml_response(user)
       end
 
-      it 'stores the SAML Request original_url in the session' do
-        expect(session[:saml_request_url]).to eq(request.original_url)
-      end
```

Comment:
the feature spec tests that controller. 

this spec is for the ` get :metadata` endpoint, which we no longer want to save this session var.